src/toil/job.py

```diff
@@ -458,6 +458,7 @@ def addToilOptions(parser):
 
         @staticmethod
         def startToil(job, options):
+            # FIXME: deprecate this function in favor of the Toil context manager
```

Comment:
You know what, why don't you just rewrite the comment instead of this FIXME. I think the Sphinx doc is reasonably explicit about the deprecation, it's just that this docstring isn't clear about it.